maven/poms/gwt/gwt-dev/pom-template.xml

```diff
@@ -114,5 +114,13 @@
             <groupId>org.eclipse.jetty</groupId>
             <artifactId>apache-jsp</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.eclipse.jdt</groupId>
+            <artifactId>org.eclipse.jdt.core</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.eclipse.jdt</groupId>
+            <artifactId>ecj</artifactId>
```

Comment:
Why add ecj here? I could understand putting batch in here, but I think that's a transitive dependency of jdt-core - and the build.xml files etc above don't ever reference ecj, nor is it in the tools patch?

Reply:
Actually batch is ecj, weird packaging from eclipse, but that is as you said is a transitive dependency and should not be included.